shortcodes/class.geojson-shortcode.php

```diff
@@ -62,6 +62,7 @@ protected function getHTML ($atts='', $content=null) {
         ?>
         <script>
             WPLeafletMapPlugin.add(function () {
+                var legendVals = new Object();
```

Comment:
It looks like the style of the project is only one `var` statement per function. So I think you should add your 'legendVals' declaration at the end of the `var` statement below. Also I find object literal declaration preferable to `new Object()`.
```
legendVals = {};
```
We use it preferably at Gaia.